service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/UploadTranslationsControllerTest.java

```diff
@@ -0,0 +1,209 @@
+
+    private static final CaseData CASE_DATA_WITH_ALL_ORDERS = CaseData.builder()
+        .reviewDraftOrdersData(ReviewDraftOrdersData.builder().build())
+        .orderCollection(List.of(element(UUID_1, GeneratedOrder.builder()
+            .type("Generated order type")
+            .dateTimeIssued(LocalDateTime.of(2020, 12, 10, 21, 2, 3))
+            .build())))
+        .standardDirectionOrder(StandardDirectionOrder.builder()
+            .dateOfUpload(LocalDate.of(2020, 12, 11))
+            .orderStatus(OrderStatus.SEALED)
+            .build())
+        .urgentHearingOrder(UrgentHearingOrder.builder()
+            .dateAdded(LocalDate.of(2020, 12, 8))
+            .build())
+        .sealedCMOs(List.of(element(UUID_2, HearingOrder.builder()
+            .status(APPROVED)
+            .dateIssued(LocalDate.of(2020, 12, 9))
+            .build())))
+        .noticeOfProceedingsBundle(List.of(element(UUID_3, DocumentBundle.builder()
+                .document(DocumentReference.builder()
+                    .filename("noticeo_c6.pdf")
+                    .build())
+                .build()
+            ), element(UUID_4, DocumentBundle.builder()
+                .document(DocumentReference.builder()
+                    .filename("noticeo_c6a.pdf")
+                    .build())
+                .build()
+            )
+        ))
+        .build();
+    private static final DynamicList RENDERED_DYNAMIC_LIST = DynamicList.builder()
+        .value(DynamicListElement.EMPTY)
+        .listItems(List.of(
+            dlElement(UUID_1, "Generated order type - 10 December 2020"),
+            dlElement(UUID_2, "Sealed case management order issued on 9 December 2020"),
+            dlElement(StandardDirectionOrder.COLLECTION_ID, "Gatekeeping order - 11 December 2020"),
+            dlElement(UUID_3, "Notice of proceedings (C6)"),
+            dlElement(UUID_4, "Notice of proceedings (C6A)"),
+            dlElement(UrgentHearingOrder.COLLECTION_ID, "Urgent hearing order - 8 December 2020")
+        )).build();
+    private static final DocumentReference TEST_DOCUMENT = DocumentReference.buildFromDocument(testDocument());
+    private static final byte[] TRANSLATED_DOC_BYTES = "TranslatedDocumentContent".getBytes();
+    private static final byte[] CONVERTED_DOC_BYTES = "ConvertedDocumentContent".getBytes();
+    private static final byte[] SEALED_DOC_BYTES = "SealedDocumentContent".getBytes();
+    private static final Document UPLOADED_TRANSFORMED_DOCUMENT = testDocument();
+
+    UploadTranslationsControllerTest() {
+        super("upload-translations");
+    }
+
+    @MockBean
+    private DocumentDownloadService documentDownloadService;
+
+    @MockBean
+    private DocumentConversionService documentConversionService;
+
+    @MockBean
+    private DocumentSealingService documentSealingService;
+
+    @MockBean
+    private UploadDocumentService uploadDocumentService;
+
+    @Test
+    void shouldDisplayAllTranslatableOrdersAboutToStart() {
```

Comment:
thoughts on splitting these tests into their own files per callback as per other controller tests

Reply:
split into files and used a helper